git/git.cpp

```diff
@@ -2735,6 +2731,7 @@ bool GitPlugin::DoExecuteCommandSync(const wxString& command, wxString* commandO
 
         clDEBUG() << "<-" << output.value() << endl;
         *commandOutput = output.value();
+#endif
```

Comment:
I assume in `!USE_SFTP`,  `IsRemoteWorkspace()` is always `false`, so no alternative code for that branch.